### DISCUSSION TITLE
error Embed twitch

### DISCUSSION THREAD

**Discussion by longjin1991** [wailsapp/wails#2005] [Q&A]
I embed twitch video to app wails, code:

\
\<div id="twitch-embed"></div>

\
\<script src="https://player.twitch.tv/js/embed/v1.js"></script>

\
\<script type="text/javascript">
  new Twitch.Player("twitch-embed", {
    video: "1621322762"
  });
\</script>

when run: wails dev.
it normal operation, show video and play it.
when run: wails build
it successful, but video twitch not show, not view.
console in devtool error:
Refused to frame 'https://player.twitch.tv/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors http://localhost:* https://localhost:* https://wails.localhost".

how to fix it when app build.
image [error:](https://imgur.com/a/T1ZEVYj)
https://imgur.com/a/T1ZEVYj

**Comment by thiswallz** [comment]
yup you cannot embed using iframes, to make this wails would need something like browserview (electron). to escape the iframe rules and embed a real view within the browser window.

**Reply by longjin1991** [reply]
thank you.
if i run command: wails dev.
it will show video and play it.
but run command: wails build.
it error: Refused to frame 'https://player.twitch.tv/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors http://localhost/:* https://localhost/:* https://wails.localhost/".
i think problem is domain: wails.localhost, how to change domain wails.localhost to localhost, example: localhost:1234?
